storybook/src/admin/form/ToggleButtonGroupField.stories.tsx

```diff
@@ -0,0 +1,350 @@
+                }}
+                subscription={{ values: true }}
+            >
+                {({ values }) => {
+                    return (
+                        <>
+                            <ToggleButtonGroupField<SampleType>
+                                label="Sample Type"
+                                name="type"
+                                options={[
+                                    {
+                                        label: "Label",
+                                        value: "label",
+                                    },
+                                    {
+                                        label: "Label 2",
+                                        value: "label-2",
+                                    },
+                                    {
+                                        label: <Info />,
+                                        value: "icon",
+                                    },
+                                    {
+                                        label: (
+                                            <Box display="flex" gap={2} alignItems="center" component="span">
+                                                <Info />
+                                                Icon and Label
+                                            </Box>
+                                        ),
+                                        value: "icon+label",
+                                    },
+                                ]}
+                            />
+
+                            <Alert title="FormState">
+                                <pre>{JSON.stringify(values, null, 2)}</pre>
+                            </Alert>
+                        </>
+                    );
+                }}
+            </FinalForm>
+        );
+    },
+};
+
+/**
+ * Sometimes it is necessary to show more options, which do not fit into one row, with the `optionsPerRow` prop it is possible to
+ * define how many options should be shown in one row.
+ */
+export const MultipleRowsExample: Story = {
+    storyName: "ToggleButtonGroupField",
+    render: () => {
+        type SampleType = "top-left" | "top-center" | "top-right" | "bottom-left" | "bottom-center" | "bottom-right";
+        interface FormValues {
+            type: SampleType;
+        }
+        return (
+            <FinalForm<FormValues>
+                initialValues={{ type: "top-left" }}
+                mode="edit"
+                onSubmit={() => {
+                    // not handled
+                }}
+                subscription={{ values: true }}
+            >
+                {({ values }) => {
+                    return (
+                        <>
+                            <ToggleButtonGroupField<SampleType>
+                                label="Sample Type"
+                                name="type"
+                                optionsPerRow={3}
+                                options={[
```

Comment:
Could we change this example to use icons instead of labels? Then we could see if it still works correctly for icons, even with the removed hardcoded dimensions.

Reply:
@manuelblum what about this?

Reply:
changed the example to showcase the FocusPoints

```
 north-west | north | north-east
 west | center | east
 south-west | south | south-east
```